src/makeCalculator.js

```diff
@@ -4,7 +4,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(callback, number) {
+      this.result = callback(this.result, number);
+
+      return this;
+    },
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
+      return a - b;
+    },
+    multiply(a, b) {
+      return a * b;
+    },
+    divide(a, b) {
+      if (b === 0) {
+        return 'erorrdivide';
```

Comment:
There is a typo in the error message returned when division by zero is attempted. It should be 'errorDivide' instead of 'erorrdivide'.